.github/workflows/gradle.yml

```diff
@@ -60,6 +60,8 @@ jobs:
           body: |
             This release should only be used if testing one of the features that have not made it to the plugin hub yet.
 
+            Note: To use this build, you must run from terminal (or command prompt): `java -ea -jar dinkplugin-1.x.x-all.jar` (be sure to specify the correct version and path to the file). Also, very recent java versions may require [additional VM options](https://github.com/runelite/runelite/wiki/Building-with-IntelliJ-IDEA#jdks-16).
+
             ${{ github.event.head_commit.message }}
```

Comment:
Can we add a note here explaining this line, I never bothered to do it for Chatterino because we have no extra body information, but it just seems like a confusing section of the body randomly saying the most recent change. Likely would be less confusing if we explained what it was